packages/core-contracts/contracts/ownership/OwnableMixin.sol

```diff
@@ -4,24 +4,38 @@ pragma solidity ^0.8.0;
 abstract contract OwnableMixin {
     error OnlyOwnerAllowed();
 
+    struct OwnerNamespace {
```

Comment:
OwnerNamespace struct and function _ownerStorage() are already defined in OwnerStorage.sol

Reply:
right, it should have been deleted!

Reply:
+1

I think OwnerStorage should be in its own file. For now it can reside just next to this file.

Reply:
fixed